<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExpertSystem</class>
 <widget class="QWidget" name="ExpertSystem">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Path of Hero</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/resources/img/icon.ico</normaloff>:/resources/img/icon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#horizontalWidget {border-image: url(:/resources/img/book.jpg)}
QWidget {
	font-family: Tempus Sans ITC;
	font-size: 15px;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="horizontalWidget" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>25</number>
      </property>
      <property name="leftMargin">
       <number>90</number>
      </property>
      <property name="topMargin">
       <number>75</number>
      </property>
      <property name="rightMargin">
       <number>90</number>
      </property>
      <property name="bottomMargin">
       <number>75</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0">
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <property name="leftMargin">
         <number>15</number>
        </property>
        <property name="topMargin">
         <number>25</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>50</number>
        </property>
        <item>
         <widget class="QLabel" name="NextEventLabel">
          <property name="maximumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="visible">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>begining_1</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="TextLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>190</height>
           </size>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Tempus Sans ITC</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="text">
           <string>Let's write a new story for your character!</string>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="PictureLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>370</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="resources.qrc">:/resources/img/WIP.jpg</pixmap>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>50</number>
        </property>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>180</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="CharacterInfo">
          <property name="font">
           <font>
            <family>Tempus Sans ITC</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="visible">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>7</number>
          </property>
          <property name="leftMargin">
           <number>15</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QRadioButton" name="radioButton_1">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>283</width>
              <height>24</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Tempus Sans ITC</family>
              <pointsize>-1</pointsize>
             </font>
            </property>
            <property name="visible">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">QRadioButton::indicator {
    width: 13px;
    height: 13px;
	image: none;
}

QRadioButton::indicator::checked {
	image: url(:/resources/img/check-mark.ico);
}
QRadioButton::indicator::unchecked::hover {
	image: url(:/resources/img/check-mark-hover.ico);
}</string>
            </property>
            <property name="text">
             <string>Yes</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <property name="autoExclusive">
             <bool>false</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_2">
            <property name="maximumSize">
             <size>
              <width>283</width>
              <height>24</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Tempus Sans ITC</family>
              <pointsize>-1</pointsize>
             </font>
            </property>
            <property name="visible">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">QRadioButton::indicator {
    width: 13px;
    height: 13px;
	image: none;
}

QRadioButton::indicator::checked {
	image: url(:/resources/img/check-mark.ico);
}

QRadioButton::indicator::unchecked::hover {
	image: url(:/resources/img/check-mark-hover.ico);
}</string>
            </property>
            <property name="text">
             <string>No</string>
            </property>
            <property name="autoExclusive">
             <bool>false</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_3">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>283</width>
              <height>24</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Tempus Sans ITC</family>
              <pointsize>-1</pointsize>
             </font>
            </property>
            <property name="visible">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">QRadioButton::indicator {
    width: 13px;
    height: 13px;
	image: none;
}

QRadioButton::indicator::checked {
	image: url(:/resources/img/check-mark.ico);
}

QRadioButton::indicator::unchecked::hover {
	image: url(:/resources/img/check-mark-hover.ico);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <property name="autoExclusive">
             <bool>false</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>70</height>
           </size>
          </property>
         </spacer>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QCheckBox" name="SaveHeroCheck">
          <property name="font">
           <font>
            <family>Tempus Sans ITC</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="visible">
           <bool>false</bool>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QCheckBox::indicator {
    width: 13px;
    height: 13px;
	image: none;
}

QCheckBox::indicator::checked {
	image: url(:/resources/img/check-mark.ico);
}
QCheckBox::indicator::unchecked::hover {
	image: url(:/resources/img/check-mark-hover.ico);
}</string>
          </property>
          <property name="text">
           <string>mark to save </string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item alignment="Qt::AlignHCenter">
           <widget class="QPushButton" name="ResetButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	background-color: qlineargradient(spread:pad, x1:0.609976, y1:0.494, x2:1, y2:1, stop:0 	rgba(180, 140, 80, 200), stop:1 rgba(255, 255, 255, 255));
	border-radius: 5px;
	border: 1px solid black;
}

QPushButton:hover {
	background-color: qlineargradient(spread:pad, x1:0.609976, y1:0.494, x2:1, y2:1, stop:0 	rgba(180, 140, 80, 255), stop:1 rgba(255, 255, 255, 255));
 }

QPushButton:pressed {
	background-color: qlineargradient(spread:pad, x1:0.609976, y1:0.494, x2:1, y2:1, stop:0 	rgba(180, 140, 80, 160), stop:1 rgba(255, 255, 255, 255));
 }
</string>
            </property>
            <property name="text">
             <string>Reset character</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QPushButton" name="SubmitButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Tempus Sans ITC</family>
              <pointsize>-1</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	background-color: qlineargradient(spread:pad, x1:0.609976, y1:0.494, x2:1, y2:1, stop:0 	rgba(180, 140, 80, 200), stop:1 rgba(255, 255, 255, 255));
	border-radius: 5px;
	border: 1px solid black;
}

QPushButton:hover {
	background-color: qlineargradient(spread:pad, x1:0.609976, y1:0.494, x2:1, y2:1, stop:0 	rgba(180, 140, 80, 255), stop:1 rgba(255, 255, 255, 255));
 }

QPushButton:pressed {
	background-color: qlineargradient(spread:pad, x1:0.609976, y1:0.494, x2:1, y2:1, stop:0 	rgba(180, 140, 80, 160), stop:1 rgba(255, 255, 255, 255));
 }
</string>
            </property>
            <property name="text">
             <string>Continue</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
